src/db_queries/admin_partners_mentors_stats.ts

```diff
@@ -21,7 +21,7 @@ export class AdminPartnersMentorStats {
     response: Response
   ): Promise<void> {
     const partnerId = request.params.partner_id;
-    const searchParameters: PartnerMentorsSearch = request.body;
+    const searchParameters: PartnerMentorsSearch = request.query;
```

Comment:
We actually want the search parameters in the body of the request.

Reply:
Hi @edmondpr, this is actually a GET request, so I updated as [by convention](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET#:~:text=It%20is%20better%20to%20just%20avoid%20sending%20payloads%20in%20GET%20requests.) we generally don't use body in http GET.
Should I convert this to POST if query doesn't work.

But let me know if I should keep it as it is. will update in frontend PR accordingly.

Reply:
You're right, I didn't take that into consideration. It would probably be best to make it a POST request in this case because it's easier to manage all the parameters in the body.